src/Service/QuoteProgressService.php

```diff
@@ -47,4 +62,9 @@ public function markAsOpened(QuoteProgress $quote): QuoteProgress
 
         return $quote;
     }
+
+    private function mailQuoteProgress(QuoteProgress $quoteProgress): void
+    {
+        Mail::to($this->data['email'])->queue($this->mailable->setQuoteProgress($quoteProgress));
```

Comment:
Is this synchronous? If so, it would be better to avoid it, and use Notification (or other, but async).

Reply:
haha, it literally says `queue`. please ignore.